web/cobrands/fixmystreet/assets.js

```diff
@@ -505,15 +480,56 @@ function get_fault_stylemap() {
     });
 }
 
+// fixmystreet.pin_prefix isn't always available here, due
+// to file loading order, so get it from the DOM directly.
+var map_data = document.getElementById('js-map-data');
+var pin_prefix = fixmystreet.pin_prefix || (map_data ? map_data.getAttribute('data-pin_prefix') : '/i/');
+
 fixmystreet.assets = {
     layers: [],
     controls: [],
 
+    stylemap_invisible: new OpenLayers.StyleMap({
+        'default': new OpenLayers.Style({
+            fill: false,
+            stroke: false
+        })
+    }),
+
+    style_default: new OpenLayers.Style({
+        fillColor: "#FFFF00",
+        fillOpacity: 0.6,
+        strokeColor: "#000000",
+        strokeOpacity: 0.8,
+        strokeWidth: 2,
+        pointRadius: 6
+    }),
+
+    style_default_select: new OpenLayers.Style({
+        externalGraphic: pin_prefix + "pin-spot.png",
+        fillColor: "#55BB00",
+        graphicWidth: 48,
+        graphicHeight: 64,
+        graphicXOffset: -24,
+        graphicYOffset: -56,
+        backgroundGraphic: pin_prefix + "pin-shadow.png",
+        backgroundWidth: 60,
+        backgroundHeight: 30,
+        backgroundXOffset: -7,
+        backgroundYOffset: -22,
+        popupYOffset: -40,
+        graphicOpacity: 1.0
+    }),
+
     selectedFeature: function() {
         return selected_feature;
     },
 
     add: function(default_options, options) {
+        if (!fixmystreet.map_type) {
```

Comment:
This breaks assets on certain cobrands (I've tried BANES, Hounslow, Bristol, Lincolnshire, Bexley. Works OK on Bucks though). It looks as though `map_type` isn't set at this point so no asset layers get added to the map.

Reply:
Okay, turns out map_type isn't always defined, I thought it was; I've switched to checking for the presence of the map div.

Reply:
Plus rewriting the code to always make sure map.js loads before assets.js. But then still back to the blank issue as elsewhere.